src/azure-cli/azure/cli/command_modules/rdbms/flexible_server_custom_postgres.py

```diff
@@ -996,6 +996,57 @@ def migration_check_name_availability(cmd, client, resource_group_name, server_n
     return get_postgresql_flexible_management_client(cmd.cli_ctx).check_migration_name_availability(subscription_id, resource_group_name, server_name, migration_name_availability_parammeters)
 
 
+def virtual_endpoint_create_func(client, resource_group_name, server_name, virtual_endpoint_name, endpoint_type, members):
+    parameters = {
+        'name': virtual_endpoint_name,
+        'endpoint_type': endpoint_type,
+        'members': [members]
+    }
+
+    return client.begin_create(
+        resource_group_name,
+        server_name,
+        virtual_endpoint_name,
+        parameters)
+
+
+def virtual_endpoint_show_func(client, resource_group_name, server_name, virtual_endpoint_name):
+
+    return client.get(
+        resource_group_name,
+        server_name,
+        virtual_endpoint_name)
+
+
+def virtual_endpoint_list_func(client, resource_group_name, server_name):
+
+    return client.list_by_server(
+        resource_group_name,
+        server_name)
+
+
+def virtual_endpoint_delete_func(client, resource_group_name, server_name, virtual_endpoint_name):
```

Comment:
THis functiona doesn't seem to be accepting yes parameter as an input. If yes is not passed it should prompt for deletion